src/Contracts/Controller/DashboardControllerInterface.php

```diff
@@ -19,6 +19,12 @@
  */
 interface DashboardControllerInterface
 {
+    /**
+     * This method will be added to the interface in EayAdmin 5.0.
+     * Not implementing it is deprecated.
+     *
+     * public static function configureControllers(EasyCorp\Bundle\EasyAdminBundle\Config\Controllers $controllers): Controllers;
+     */
```

Comment:
You can move this comment to above the interface adding the `@method` tag so IDEs can autocomplete that already. The return type is in front. This is also done in https://github.com/SymfonyCasts/verify-email-bundle/blob/main/src/VerifyEmailHelperInterface.php for instance.
```suggestion
/**
 * This method will be added to the interface in EayAdmin 5.0.
 * Not implementing it is deprecated.
 *
 * @method static EasyCorp\Bundle\EasyAdminBundle\Config\Controllers configureControllers(EasyCorp\Bundle\EasyAdminBundle\Config\Controllers $controllers);
 */
```

Reply:
Thanks! Luckily, we can get rid of this method because we now solve this feature via the new `#[AdminDashboard]` attribute. But I will keep this in ind for future needs. Thank you.